File: codeforces/6.bit++/bit++.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  unsigned short int n;
  cin >> n;
  if (n >= 1 && n <= 150) {
    int x = 0;
    string statement;
    while (n--) {
      cin >> statement;
      if (statement[1] == '+') {
        x++;
      }
      else if (statement[1] == '-') {
        x--;
      }
      else {
        continue;
      }
    }
    cout << x << endl;
  }
  return 0;
}

File: codeforces/low_difficulty/800_points/6.bit++_[282A]/bit++.py
# Author: José Rodolfo (jric2002)
n = int(input())
if (n >= 1 and n <= 150):
  x = 0
  for i in range(0, n):
    statement = str(input())
    if (statement[1] == '+'):
      x += 1
    elif (statement[1] == '-'):
      x -= 1
    else:
      continue
  print(x)